package.json

```
@@ -54,11 +54,11 @@
     "lint-staged": "^10.5.4",
     "prettier": "~2.8.3",
     "redux": "^4.2.0",
-    "rollup": "~2.40.0",
+    "rollup": "~3.28.1",
     "strip-ansi": "^6.0.1"
   },
   "peerDependencies": {
-    "rollup": "^2.0.0"
+    "rollup": "^2.0.0 || ^3.0.0"
```

I would like to drop rollup v2 support for now.

At this point test suite works with more recent rollup v2 such as 2.40 & 2.79.1 but not all the way back to `2.0.0`. And the proposed updates will succeed with Rollup pre-release `4.0.0-13`. So I would likely favor moving forward supporting the newer Node.js versions.

```suggestion
    "rollup": "^3.0.0"
```